bibliopixel/remote/control.py

```diff
@@ -57,10 +58,17 @@ def __init__(self, config, animations):
             anim_cfg = dict(DEFAULT_ANIM_CONFIG)
             anim_cfg.update(anim)
             if not anim_cfg['display']:
-                anim_cfg['display'] = anim['name']
+                anim_cfg['display'] = anim_cfg['name']
             anim_cfg['name'] = ''.join(e for e in anim['name'] if e.isalnum())
-            anim_cfg['animation'].on_completion = self.on_completion
-            self.animation_objs[anim_cfg['name']] = anim_cfg['animation']
+
+            if anim_cfg['animation'] is None:
+                anim_cfg['display'] = 'LOAD FAILED: ' + anim_cfg['display']
+                anim_cfg['valid'] = False
+                anim_cfg['bgcolor'] = 'rgb(16, 16, 16)'
```

Comment:
Note to future me: I think this value is too light, it's basically black on screen. Maybe 48